code/modules/mob/living/carbon/human/human_mob.dm

```diff
@@ -1962,3 +1962,68 @@ Eyes need to have significantly high darksight to shine unless the mob has the X
 /mob/living/carbon/human/proc/apply_offstation_roles(source)
 	SIGNAL_HANDLER
 	mind.offstation_role = TRUE
+
+/mob/living/carbon/human/proc/expert_chef_knowledge()
```

Comment:
Spooks me perfwise, can you check how long this function takes with like 20 valid objects?